### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by Le-Caignec**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
+    "plugin:sonarjs/recommended",
+    "plugin:jest/recommended"
+  ],
+  "rules": {
+    "import/no-extraneous-dependencies": [
+      "error",
+      {
+        "devDependencies": true
+      }
+    ],
```
Comment (PierreJeanjacquot) [inline .eslintrc.json:22]: move this in override for tests

File: package.json
Comment (PierreJeanjacquot) [inline package.json:1]: useless `--ext .ts` in script `lint`

**Review by PierreJeanjacquot — CHANGES_REQUESTED** [review]
see comments

File: package.json
Comment (Le-Caignec) [reply package.json:1]: Commit: aa51653d4a02edc86449c4ecbea8586d8c878e64

**Review by PierreJeanjacquot — APPROVED** [review]
:rocket:

**Pull request merged**